ci/jenkinsfiles/build.groovy

```diff
@@ -314,7 +314,7 @@ pipeline {
               nxKube.helmGetPreviewLogs("${PREVIEW_NAMESPACE}")
               cucumber (
                 fileIncludePattern: '**/*.json',
-                jsonReportDirectory: "${FRONTEND_FOLDER}/ftest/target/cucumber-reports/",
```

Comment:
not needed and the 
![Screenshot 2021-07-01 at 14 32 24](https://user-images.githubusercontent.com/59877775/124124806-2dd89a00-da79-11eb-9362-715f8545223a.png)

Reply:
to have the screenshot report update https://github.com/nuxeo/nuxeo-coldstorage/blob/NXP-30487-fixColdstorageFtestsPipeline/ci/jenkinsfiles/build.groovy#L322

```
//we can't use full path when archiving artifacts
artifacts: 'nuxeo-coldstorage-web/target/**, 'nuxeo-coldstorage-web/ftest/target/** logs/*.log'
```